ddtrace/llmobs/_evaluations/ragas/faithfulness/evaluator.py

```diff
@@ -0,0 +1,85 @@
+import atexit
+from concurrent import futures
+import math
+import time
+from typing import Dict
+from typing import Optional
+
+from ddtrace import config
+from ddtrace.internal import forksafe
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.periodic import PeriodicService
+
+from ...._writer import LLMObsEvaluationMetricEvent
+
+
+logger = get_logger(__name__)
+
+
+class RagasFaithfulnessEvaluator(PeriodicService):
```

Comment:
Does it make more sense to make a base RagasEvaluator class and then extend that with custom evaluate() functions?

Reply:
I was planning on doing that after more ragas evaluators were introduced (e.g. how we refactored the span writer into base writer + span writer + eval metric writer) - i didn't see the value of doing that now with only faithfulness